scripts/plop-templates/package/package.json.hbs

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.6.2",
-    "@emotion/core": "^10.1.1",
+    "emotion": "^10.0.17",
```

Comment:
A pedantic nit here, if we don't sort this alphabetically yarn will do it for us and create a diff on the first run, which adds noise.

Reply:
updated 🧐